### PR TITLE
Initial website content

### LINKED ISSUE #10: Use the official logo on the website.
The logo for the light mode is not the official logo
image

should be this one:
image

here's the official brand and logo kit:
[Overte-Brand_and_Logo-Kit.zip](https://github.com/overte-org/overte-website/files/8158574/Overte-Brand_and_Logo-Kit.zip)

### PR DESCRIPTION
**Pull request by ksuprynowicz**
Probably still needs work, but I'm stuck and I need help with this.

### CODE REVIEW

File: src/content-blocks/contact.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  name: Overte e.V.
+  date: 05.02.2022
+tags: contentBlock
+articleTopLinks: {
+  
+}
+---
+
+
+Join us on our [Matrix](https://matrix.to/#/#overte:matrix.org) and [Discord](https://discord.gg/4YuQvc8K2f)
```
Comment (JulianGro) [inline src/content-blocks/contact.md]: I don't think we should advertise Discord.
It is proprietary, insecure, and will break compatibility again eventually. For all we know, they might even disallow bridging in the future.

**Review by JulianGro — CHANGES_REQUESTED** [review]
Please do not commit build files. This will double the size of the repository and it's changes. It might also catch someone reviewing PRs off guard and have them review the built files (like it just did with me).

File: src/content-blocks/contact.md (hunk 1 above)
Comment (ksuprynowicz) [reply src/content-blocks/contact.md]: I have no idea how to solve this. Most of my friends don't and won't have Matrix. I'm tired of begging again and again for keeping this.

Comment (ksuprynowicz) [reply src/content-blocks/contact.md]: I don't want to exclude people from already small community.

Comment (ksuprynowicz) [reply src/content-blocks/contact.md]: I'm giving up on this for now.

Comment (ksuprynowicz) [reply src/content-blocks/contact.md]: I added a statement promoting Matrix, written by Kreolis.
It's a reasonable compromise in my opinion. Let me know what you think about it.

Comment (ksuprynowicz) [reply src/content-blocks/contact.md]: I also removed the build directory

Comment (JulianGro) [reply src/content-blocks/contact.md]: I don't see a statement. Maybe you forgot to commit it?
How about something like this:
```suggestion
Join us on [Matrix](https://matrix.to/#/#overte:matrix.org) or on our [Discord bridge](https://discord.gg/4YuQvc8K2f)
```

Comment (ksuprynowicz) [reply src/content-blocks/contact.md]: @JulianGro 
It's here:
https://github.com/overte-org/overte-website/pull/8/commits/42733a05475c87a3c87b966d67b74920c1244593

**Comment by JulianGro** [comment]
> (...) I need help with this

What exactly do you need help with?

**Comment by ksuprynowicz** [comment]
> What exactly do you need help with?

I'm not sure what else we need to add to have minimum viable website. I will also add more images to gallery tomorrow morning and image thumbnail for initial news article.

**Comment by Kreolis** [comment]
Appimage and installer download links are broken

**Pull request merged**